### PR TITLE
fixups from 1es migration

### PR DESCRIPTION
**Pull request by benbp**

### CODE REVIEW

**Review by alzimmermsft — APPROVED** [review]

File: sdk/spring-experimental/ci.yml [hunk 1]
```diff
@@ -1,11 +0,0 @@
-# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
-
-trigger: none
```
Comment (weshaggard) [inline sdk/spring-experimental/ci.yml:3]: Do we need to delete this pipeline in devops as well?

**Pull request merged**